documentation/docs/steps/gctsExecuteABAPUnitTests.md

```diff
@@ -0,0 +1,41 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+This step will execute every unit test associated with a package belonging to the specified local repository on an ABAP system.
```

Comment:
This duplicates the `longDescription` which is pasted into `${docGenDescription}`. I would suggest to add the "Learn more.. " part to the `longDescrioption` in the YAML and remove this part.

Reply:
I adjusted it a bit. I would like to keep it under prerequisites.